### PR TITLE
PW-859 remove documents when PASSED is received from Adyen.

### PR DESCRIPTION
**Pull request by rikterbeek**
For documents in test mode always add description passed to get account in payout modus, as well needed to test passed notification checks. Add account in tier1 for receiving passed notifications for documents. Improved the helper to retrieve the right notifications for a shareholder.

### CODE REVIEW

File: src/test/java/com/adyen/mirakl/cucumber/stepdefs/IdentityVerificationSteps.java [hunk 1]
```diff
@@ -218,7 +218,7 @@ public void theUpdatedDocumentsAreSuccessfullyUploadedToAdyen(DataTable table) t
         }
     }
 
-    @Then("^the documents will be removed for each of the UBOs$")
+    @Then("^the documents are removed for each of the UBOs$")
     public void theDocumentsWillBeRemovedForEachOfTheUBOs() {
```
Comment (Aleffio) [inline src/test/java/com/adyen/mirakl/cucumber/stepdefs/IdentityVerificationSteps.java]:
```suggestion
    public void theDocumentsAreRemovedForEachOfTheUBOs() {
```

File: src/test/java/com/adyen/mirakl/cucumber/stepdefs/IdentityVerificationSteps.java [hunk 2]
```diff
@@ -232,4 +232,14 @@ public void theDocumentsWillBeRemovedForEachOfTheUBOs() {
     public void eachUBOWillReceiveARemedialEmail(String title) throws Throwable {
         validationCheckOnReceivedEmails(title, shop);
     }
+
+
+    @And("^the shop is updated to tier1$")
+    public void updateShopToTier3() throws Throwable {
```
Comment (Aleffio) [inline src/test/java/com/adyen/mirakl/cucumber/stepdefs/IdentityVerificationSteps.java]:
```suggestion
    public void theShopIsUpdatedToTier1() throws Exception {
```

Is _throws Throwable_ necessary or can we use Exception?

File: src/test/resources/adyenRequests/COMPANY_VERIFICATION_DATA_PROVIDED.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
         "accountHolderCode": "11141",
         "shareholderCode": "c6adfbe1-4794-4e31-9861-9f69dee3a60e",
         "statusSummaryItems": [],
-        "verificationStatus": "DATA_PROVIDED",
+        "verificationStatus": "PASSED",
```
Comment (Aleffio) [inline src/test/resources/adyenRequests/COMPANY_VERIFICATION_DATA_PROVIDED.json]: let's change the file name or create a new one

Comment (rikterbeek) [reply src/test/resources/adyenRequests/COMPANY_VERIFICATION_DATA_PROVIDED.json]: fixed renamed the filename

**Pull request merged**